src/os/bluestore/BitAllocator.h

```diff
@@ -143,20 +143,27 @@ class BmapEntry {
 
 private:
   bmap_t m_bits;
-  static bool m_bit_mask_init;
-  static bmap_mask_vec_t m_bit_to_mask;
-
-
-  static void _init_bit_mask();
 
 public:
   MEMPOOL_CLASS_HELPERS();
-  static bmap_t full_bmask();
-  static int64_t size();
-  static bmap_t empty_bmask();
-  static bmap_t align_mask(int x);
-  static bmap_t bit_mask(int bit_num);
-  bmap_t atomic_fetch();
+  static bmap_t full_bmask() {
```

Comment:
@varadakari : No. Inline is unneded in declaration. It could not be inline in declaration because it will be local to compilation unit. (some of out unit tests will not work)
http://stackoverflow.com/questions/9734175/why-are-class-member-functions-inlined
"A member function may be deﬁned (8.4) in its class deﬁnition, in which case it is an inline member function (7.1.2) ..."
http://www.lcdf.org/c%2B%2B/clause9.html